test/integration/GCHP/integrationTestExecute.sh

```diff
@@ -4,7 +4,7 @@
 #SBATCH -N 1
 #SBATCH -t 0-5:00
 #SBATCH -p REQUESTED_PARTITION
-#SBATCH --mem=90000
+#SBATCH --mem=150000
```

Comment:
Was something running out of memory? We shouldn't need this much for c24 runs. Is it for TOMAS?

Reply:
@lizziel: I think it might be for TOMAS.  I can try reducing it somewhat to see if it still runs with a lower amount.

Reply:
@lizziel: I just looked at the no-diff-to-benchmark branch.  This is the current GCHP file header:
```console
#!/bin/bash

#SBATCH -n 24
#SBATCH -N 1
#SBATCH -c 1
#SBATCH -t 0-5:00
#SBATCH -p sapphire,huce_cascade,seas_compute,shared
#SBATCH --mem=90000
#SBATCH --mail-type=END
```

Reply:
That is an update I added in the last commit (https://github.com/geoschem/geos-chem/pull/1992/commits/e927528f7fe1f7c7b40768fd3c05626c0b822749). From the commit message:

>Also increased memory requested in GCHP integration test to avoid out-of-
memory errors.